static/js/textMarkInfoDialog.js

```diff
@@ -265,6 +265,18 @@ textMarkInfoDialog.prototype._selectTextOfRepPosition = function(getRepPosition,
   });
 };
 
+textMarkInfoDialog.prototype.eventTargetIsATextMarkInfoDialog = function(e) {
+  return this._elementIsATextMarkInfoDialog(e.target);
+};
+
+textMarkInfoDialog.prototype._elementIsATextMarkInfoDialog = function(element) {
+  var infoDialog = this.infoDialog.widget.get(0);
+  var editDialog = this.editDialog.widget.get(0);
+  var elementIsInsideInfoDialog = $(element).closest(infoDialog).length !== 0;
+  var elementIsInsideEditDialog = $(element).closest(editDialog).length !== 0;
+  return elementIsInsideInfoDialog || elementIsInsideEditDialog;
+};
+
```

Comment:
parece que [dá para usar](https://stackoverflow.com/a/3028037/7884942) tb `this.*Dialog.widget.contains(element)`, se não quiser usar jQuery